build/make/deploy-debian.mk

```diff
@@ -1,4 +1,7 @@
 # This Makefile holds all targets for deploying and undeploying
+# Uses the variable APT_REPO to determine which apt repos should be used to deploy
+# -> APT_REPO=ces-premium results in a deploy to the premium apt repository
+# -> Everything else results in a deploy to the public repositories
```

Comment:
I think this explanation is of better assistance if the user can read it in the main Makefile